website/docs/index.html.markdown

```diff
@@ -43,10 +43,26 @@ The following arguments are supported:
 * `enable_ddi` - (Optional) This sets the permission schema to a DDI-compatible schema. 
 Users of the managed SaaS product should not need to set this.
 Users of DDI should set this to true if managing teams, users, or API keys through this provider.
-* `rate_limit_parallelism` - (Optional) Integer for parallelism amount
-  (terraform's default is 10). If set, uses an alternate strategy to handle
-  rate limiting from the NS1 API. Give this a try if you are processing a large
-  number of resource and running into `429` rate-limiting errors.
+* `rate_limit_parallelism` - (Optional) Integer for parallelism amount (terraform's default is 10).
+    NS1 uses a token-based method for rate limiting API requests. Details of which can be found here: https://help.ns1.com/hc/en-us/articles/360020250573-About-API-rate-limiting.
+    
+    By default, the NS1 provider uses the standard strategy of the underlying [NS1 Go SDK](https://github.com/ns1/ns1-go) for handling the NS1 API rate limit:
+    an operation waits after every API request for a time equal to the rate limit period of that request type divided by the corresponding tokens renaming.
```

Comment:
i thought the default strategy was "noop"?

Reply:
yeah, the default in the sdk is noop, but the terraform provider default sets it to the sleep strategy, I'll clarify the wording, good call.